frontend/src/features/board/BoardSlice.tsx

```diff
@@ -146,6 +156,16 @@ export const slice = createSlice({
       state.detailError = undefined;
       state.detailLoading = false;
     });
+    builder.addCase(deleteBoard.fulfilled, (state, action) => {
+      const indexOfDeletedBoard = state.all.findIndex((obj) => {
+        if (obj.id == action.payload) {
+          return true;
+        }
+        return false;
+      });
```

Comment:
This can be kept short
```suggestion
      const indexOfDeletedBoard = state.all.findIndex(board => board.id === action.payload);
```

Reply:
Thank you @rrebase 

The type of `board.id` is number while the type of `action.payload` and return value of `deleteBoard` is string.

So [a new commit](https://github.com/rrebase/knboard/pull/73/commits/eb504668e3c76ee3bf13b58cd58f0f0021582a39) make all of them above be restricted to be `string` like [fetchUserById  typing-createasyncthunk](https://redux.js.org/usage/usage-with-typescript#typing-createasyncthunk) does.

`const { id } = useParams();` is used which makes the type of `id` is `any`. Actually, it passes a `string` type to `Id`/`number` type. That's why `dispatch(deleteBoard(id));` works before.

Let me know if you have any concern.